.travis.yml

```diff
@@ -1,4 +1,20 @@
-language: java
+language: android
 jdk:
   - openjdk7
   - oraclejdk8
+env:
+  - TEST=java
+  - TEST=android
+matrix:
+  exclude:
+    - jdk: oraclejdk8
+      env: TEST=android
+android:
+  components:
+    - android-15
+    - build-tools-21.1.1
```

Comment:
It's ok to use this build-tools. But it would be better to use up-to-date tools which are 23.0.2 if it's possible in Travis 😊